manifests/integreatly-cloud-resources/cloud-resources-0.8.0/cloud-resources.v0.8.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,374 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Unless this has changed recently, the history of CSVs should only be kept in the operator repo. So I think this version can be deleted